src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM_SAIL = 20;
+  const LONG_TERM_SAIL = 50;
+  const CAR_RENT = 40;
```

Comment:
The variable name 'CAR_RENT' could be more descriptive. Consider renaming it to 'DAILY_RENT_COST' to accurately reflect its purpose.